Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocCheckBox.css

```diff
@@ -3,38 +3,64 @@
 .bocCheckBox
 {
   margin-right: 0.5em;
+  white-space: nowrap;
+}
+
+.bocCheckBox input
+{
+  position: absolute;
+  z-index: -1;
+}
+
+.bocCheckBox img
+{
+  margin: 2px;
   vertical-align: middle;
+  height: 16px;
+  width: 16px;
+}
+
+.bocCheckBox input + span::before
+{
+  margin: 2px;
+  visibility: visible;
+  background-image: url('../Image/sprite.svg#CheckBoxFalse');
+  width: 16px;
+  height: 16px;
   display: inline-block;
-  white-space: nowrap;
+  content: "";
+  z-index: 1;
+  position: relative;
 }
 
-.bocCheckBox.readOnly
+.bocCheckBox input[checked] + span::before
 {
+  background-image: url('../Image/sprite.svg#CheckBoxTrue');
 }
 
-.bocCheckBox.disabled
+.bocCheckBox,
+.bocCheckBox > input + span
 {
+  display: flex;
+  align-items: center;
 }
 
-.bocCheckBox input
+/* Fixes for element focus */
+
+.bocCheckBox span[role=checkbox]:focus
```

Comment:
About the outline: since Chrome and Mozilla support focus-within, should we go all-in and draw the focus on checkbox and description at the same time or defer this until later?

Reply:
Imo it would be best to do things like this - affecting multiple controls - at the end.

Reply:
just to be clear, this doesn't refer to multiple controls, just the boccheckbox elements (input +span just only input).

Reply:
Yes, focus handling does affect more of them tho, and deciding on how the focus should be drawn might be seen best when the theme is done.

Reply:
This true, but since we're custom-drawing anyway in BocCheckbox, we could go either way. But waiting's okay for me.